pkg/rhel/openvdc.spec

```diff
@@ -130,6 +134,11 @@ if [ -d /etc/mesos-slave ]; then
     cp -p /opt/axsh/openvdc/share/mesos-slave/attributes.lxc /etc/mesos-slave/attributes
   fi
 fi
+if [ -d /usr/share/lxc/templates ]; then
+  if [ ! -f /usr/share/lxc/templates/lxc-openvdc ]; then
+    cp /opt/axsh/openvdc/share/lxc-templates/lxc-openvdc /usr/share/lxc/templates/lxc-openvdc
+  fi
+fi
 
```

Comment:
``openvdc-executor-lxc`` RPM has dependency to ``lxc-template`` RPM. you don't need to check the ``/usr/share/lxc/templates`` path. Just make a symbolic link or install the ``lxc-openvdc`` binary to the path.